packages/design-system/src/styles/components/_Autocomplete.scss

```diff
@@ -1,60 +1,37 @@
 @use '../layout' as *;
 @use '../mixins';
+@use './Dropdown' as *;
+
+// Autocomplete styles extend Dropdown styles
 
 .ds-c-autocomplete {
-  max-width: var(--form__max-width);
-  position: relative;
-  // Undoing the clearfix class on the div containing the label element
-  .ds-u-clearfix {
-    &::before {
-      content: none;
-      display: block;
-    }
-    &::after {
-      clear: none;
-      content: none;
-      display: block;
-    }
-  }
-  input:focus {
-    @include mixins.focus-styles-position;
-  }
+  @extend .ds-c-dropdown;
 }
 
 .ds-c-autocomplete__label {
   margin: 0;
   padding: $spacer-1;
 }
 
-.ds-c-autocomplete__list {
-  background-color: var(--autocomplete__background-color);
-  border: 1px solid var(--autocomplete__border-color);
-  max-height: 320px;
-  overflow-y: auto;
-  padding: $spacer-1;
-  position: absolute;
-  width: 100%;
-  z-index: var(--z-dialog);
+.ds-c-autocomplete__menu-container {
+  @extend .ds-c-dropdown__menu-container;
 }
 
-.ds-c-autocomplete__list-item {
-  color: var(--autocomplete-item__font-color);
-  padding: $spacer-1;
+.ds-c-autocomplete__menu {
+  @extend .ds-c-dropdown__menu;
 }
 
-.ds-c-autocomplete__list-item--active {
-  background-color: var(--autocomplete-item__background-color--active);
-  color: var(--autocomplete-item__font-color--active);
+// Experienced an error using `@extend .ds-c-dropdown__menu-item` where it wouldn't compile styles for this class.
+// I'm assuming the cause is due to the extended class rules are also applied to `.ds-c-dropdown__menu-item-group` (.ds-c-dropdown__menu-item, .ds-c-dropdown__menu-item-group {}). When styles are split into individual rules, it works, but is less DRY. Attempted using the % placeholder selector, but that didn't work either.
+.ds-c-autocomplete__menu-item {
+  padding: var(--text-input__padding);
```

Comment:
I don't think `@extend` not working here is a blocker since it's using a token all themes have. The purpose of this class is to ensure the list items have the same padding as the text fields in a given theme and this token solves that.